rmgpy/cantherm/qchem.py

```diff
@@ -173,7 +181,9 @@ def loadGeometry(self):
                 number.append('17')
             else:
                 raise NotImplementedError('Atomic atom {0:d} not yet supported in loadGeometry().'.format(atom[i]))
-        number = numpy.array(number, numpy.int)       
+        number = numpy.array(number, numpy.int)
+        if number == [] or coord == [] or mass == []:
```

Comment:
I suggest just checking `if not (number and coord and mass)` instead of `if number == []`, etc. since `if number` evaluates to `False` if `number` is an empty list.

Reply:
This is a little more complicated because if the arrays are not empty then it is ambiguous to ask their truth value (which a python error message was kind enough to tell me). I might could make it work by using a combination of any() calls, but then it seems like beyond what we're asking it.

Reply:
Oh right, I forgot that happens with numpy arrays. Maybe we should instead check if `len(array) == 0`. Mostly I just brought this up because it seems a little weird to me to compare a list to an array.